x-pack/plugins/security_solution/server/search_strategy/security_solution/factory/hosts/risk_score/query.hosts_risk.dsl.ts

```diff
@@ -5,30 +5,42 @@
  * 2.0.
  */
 
-import { Direction, HostsRiskScoreRequestOptions } from '../../../../../../common/search_strategy';
+import {
+  Direction,
+  HostsRiskScoreRequestOptions,
+  HostRiskScoreFields,
+  HostRiskScoreSortField,
+} from '../../../../../../common/search_strategy';
 
-const QUERY_SIZE = 10;
+import { createQueryFilterClauses } from '../../../../../utils/build_query';
+
+export const QUERY_SIZE = 10;
 
 export const buildHostsRiskScoreQuery = ({
   timerange,
   hostNames,
+  filterQuery,
   defaultIndex,
-  limit = QUERY_SIZE,
-  sortOrder = Direction.desc,
+  pagination: { querySize, cursorStart } = {
+    querySize: QUERY_SIZE,
+    cursorStart: 0,
+  },
+  sort,
 }: HostsRiskScoreRequestOptions) => {
-  const filter = [];
-
-  if (timerange) {
-    filter.push({
-      range: {
-        '@timestamp': {
-          gte: timerange.from,
-          lte: timerange.to,
-          format: 'strict_date_optional_time',
-        },
-      },
-    });
-  }
+  const filter = createQueryFilterClauses(filterQuery);
+
+  // @pablo we should probably stop sending it too
```

Comment:
I believe that we need to filter by timerange on the host detail page. I am uncommenting this code.